kong/conf_loader/init.lua

```diff
@@ -541,6 +541,7 @@ local CONF_INFERENCES = {
   untrusted_lua_sandbox_environment = { typ = "array" },
 
   legacy_worker_events = { typ = "boolean" },
+  wrpc_hybrid_protocol = { typ = "boolean" },
```

Comment:
Should we remove the `legacy_worker_events` switch?

Reply:
Do we have a test for this switch?

Reply:
`wrpc_hybrid_protocol` is only for wrpc test, it will not be enabled in the field.